lib/rack/tracker/handler.rb

```diff
@@ -56,4 +56,28 @@ def write_event(event)
   def handler_name
     self.class.name.demodulize.underscore
   end
+
+  def tracker_options
+    @_tracker_options ||= {}.tap do |tracker_options|
+      options.slice(*self.class::ALLOWED_TRACKER_OPTIONS).each do |key, value|
+        if option_value = value.respond_to?(:call) ? value.call(env) : value
+          tracker_options[tracker_option_key(key)] = tracker_option_value(option_value)
+        end
+      end
+    end
+  end
```

Comment:
looks good!! should also add some specs for that on the handler level.

Reply:
Yep, I'll try to come up with some tests for this. Thanks for the feedback!